libzeth/core/bits.cpp

```diff
@@ -62,4 +62,13 @@ std::vector<bool> bit_vector_from_size_t_be(size_t x)
     return res;
 }
 
+void bit_vector_write_string(const std::vector<bool> &bits, std::ostream &out_s)
```

Comment:
I'd rename to `bit_vector_write` for consistency with other serialization functions (e.g. https://github.com/clearmatics/libff/blob/develop/libff/algebra/fields/field_serialization.hpp#L33)

Reply:
Where the serialization functions use a specific encoding, the convention is `<type>_<operation>_<encoding>`.  In libff, we have functions which are parameterized by the encoding, hence we drop the `_<encoding>` part in the name.

Reply:
Ah yes indeed, thanks